ambry-store/src/main/java/com/github/ambry/store/CompactionPolicyCounter.java

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.store;
+
+import com.github.ambry.config.StoreConfig;
+import java.util.concurrent.atomic.AtomicInteger;
+
+
+/**
+ * A counter used to switch {@link CompactAllPolicy}.
+ */
+public class CompactionPolicyCounter {
```

Comment:
nit: drop public and make this class package-private.

Reply:
Updated.